pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 
 dependencies:
   async: '>1.9.0 <3.0.0'
-  protobuf: '>=0.10.4 <0.11.0'
```

Comment:
It looks like these changes are backwards compatible, can we leave it at `>=0.10.0 <0.12.0`?

Reply:
good point - we do not have an `@override` annotation. On the added methods.

Fixed.